notificationapi_android_sdk/src/main/java/com/notificationapi/notificationapi_android_sdk/models/NotificationApiDeviceInfo.kt

```diff
@@ -0,0 +1,21 @@
+package com.notificationapi.notificationapi_android_sdk.models
+
+import android.content.Context
+
+class NotificationApiDeviceInfo internal constructor(context: Context) {
+    val app_id: String
+    val ad_id: String
+    val device_id: String
+    val platform: String
+    val manufactuer: String
+    val model: String
+
+    init {
+        app_id = context.packageName
+        ad_id = ""
+        device_id = ""
```

Comment:
`ad_id` and `device_id` both require tasks/futures to get the value. I decided to skip this complexity for now